buildwin.ps1

```diff
@@ -505,8 +505,6 @@ function Build
 	else										 { $extension = 'vcxproj' }
 
 	$platformName = ''
```

Comment:
It seems that `$platformName` can be completely removed?

Reply:
mmh yes i think so i will remove it, test it and then push the changes